app/services/analytics_events.rb

```diff
@@ -4609,15 +4609,13 @@ def reactivate_account_visit
   # @param [String] validator_class Class name of validator
   # @param [String, nil] exception_class Class name of exception, if error occurred
   # @param [String, nil] phone_country_code Country code associated with reCAPTCHA phone result
-  # @param [String] recaptcha_version
```

Comment:
seems kind of useful to keep this, like even if all new requests are the same version going forward, the comparing new to old would be clearer?

Reply:
Yeah, maybe I could hard-code it for the short-term, especially since we have some dashboards which query based on the version. I don't see it being necessary to keep long-term though. Another thought here (which also touches on your later comment about renaming the configuration) is that reCAPTCHA Enterprise doesn't have this notion of v2 and v3 (it calls them ["score-based"](https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages) and ["checkbox"](https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages-with-checkbox)), so the versioning may end up becoming confusing.

Reply:
ah I didn't realize we were switching away from the versioned API entirely, makes sense to remove. maybe the version is just "enterprise" then instead of `v2` or `v3`

Reply:
Technically we still support the free version of reCAPTCHA for local testing, which still references versioning when setting up keys. But we'll only be using the invisible checks ("score-based" in Enterprise or "v3" in reCAPTCHA Free). I do think it makes sense to name things around how it behaves in deployed environments, which is Enterprise.